### PR TITLE
feat(lists): Add support for `keyboardShouldPersistTaps`

### PR DESCRIPTION
**Pull request by reshavsingla**
Add support for `keyboardShouldPersistTaps` to List and SectionList. 

| List  | SectionList  |
|---|---|
| list  |  sectionlist |

### CODE REVIEW

File: examples/ui/ui-elements/list/keyboard-persist-tap/index.xml.njk [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    <modifier focused="true">
+    <style borderBottomColor="#4778FF"/>
+    </modifier>
+  </style>
+{% endblock %}
+
+{% block container %}
+  <list keyboard-should-persist-taps="handled">
+    <item key="field" style="item">
+      <text-field name="field" style="input" auto-focus="true" placeholder="Tap!" />
```
Comment (flochtililoch) [inline examples/ui/ui-elements/list/keyboard-persist-tap/index.xml.njk:32]: We now have the ability to create sudo component to abstract common XML markup, these live under `examples/_includes/` and can be included using the syntax `{% from ... import ... %}`.
Would you mind abstracting the text field and accompanying styles as a macro?

**Review by flochtililoch — COMMENTED** [review]
Second thought on this, I wonder if we should make `handled` the default value to match how scroll views are setup. In theory this shouldn't affect existing usage of section lists, and we would still have a server side way to solve any issue if needed with the new argument.

**Comment by reshavsingla** [comment]
> Second thought on this, I wonder if we should make `handled` the default value to match how scroll views are setup. In theory this shouldn't affect existing usage of section lists, and we would still have a server side way to solve any issue if needed with the new argument.

Making the default as `handled` might break the intended use for the community. So, I would argue supporting the current default behavior and add ability to override it when needed through pros.

File: examples/ui/ui-elements/list/keyboard-persist-tap/index.xml.njk (hunk 1 above)
Comment (reshavsingla) [reply examples/ui/ui-elements/list/keyboard-persist-tap/index.xml.njk]: Addressed in https://github.com/Instawork/hyperview/pull/940/commits/f366c4edd5b2da8cf7585c15b65cdb533070b3f6

**Review by flochtililoch — APPROVED** [review]

**Pull request merged**